### ISSUE TITLE
Custom 404 Image

### ISSUE THREAD

**Issue by denzelwamburu** [wallin/angular-gravatar#38]
Great Project here :smiley: 

Any directions or clue on how to use your custom image when the email provided is not available?

**Comment by wallin** [comment]
Hi! Thanks.

Yes, you can set the default parameter (https://en.gravatar.com/site/implement/images/) by settings the  `gravatar-default` in the directive or for all images through the provider config:

```js
    gravatarServiceProvider.defaults = {
      size     : 100,
      "default": 'http://url.jpg'
    };
```

**Comment by denzelwamburu** [comment]
Great Thanks.

**Comment by daroczig** [comment]
Probably I miss something here, but it seems to me that the `default` value is appended after `https://i0.wp.com/` -- so cannot specify a custom hosted image with this approach. Can someone please provide some pointers on how to achieve that?

**Comment by thewarpaint** [comment]
@daroczig Can you post a minimal case where you see this behaviour? Are you specifying an absolute URL, i.e., including the domain?

**Comment by daroczig** [comment]
Thanks, @thewarpaint - you helped me a lot :)

While creating a minimal repro example at https://jsfiddle.net/daroczig/ADukg/16246/
It turned out that it works with absolute URL including a real domain ... I was testing locally with relative path and also using absolute path starting with http://localhost:8000 etc -- which did not work. 

tl;dr -- all good, sorry for the noise
